src/com/facebook/buck/apple/BuiltinApplePackage.java

```diff
@@ -46,16 +51,27 @@
   private final Path temp;
   private final BuildRule bundle;
 
+  @AddToRuleKey
+  private final Optional<Tool> swiftStdlibTool;
```

Comment:
Should we just grab this off the underlying `AppleBundle` instead (which would remove the change to the file above)?